Spanish guide for CCS811 Firmware Update

erguro1973 opened the pull request:
[Procedimiento para la actualización de firmwareCSS811_1.1.pdf](https://github.com/erguro1973/CCS811/files/5407858/Procedimiento.para.la.actualizacion.de.firmwareCSS811_1.1.pdf)
Spanish guide for CSS811 Firmware Update thanks Maarten for your great job!

maarten-pennings commented:
Hi Ernesto,

Thanks, I have added the spanish doc.
I did rename it slightly (underscores), and put it in a subdir, but I have
also added a link from the main readme.

regards,

  Maarten


On Tue, Oct 20, 2020 at 1:38 PM Ernesto <notifications@github.com> wrote:

> Procedimiento para la actualización de firmwareCSS811_1.1.pdf
> <https://github.com/erguro1973/CCS811/files/5407858/Procedimiento.para.la.actualizacion.de.firmwareCSS811_1.1.pdf>
> Spanish guide for CSS811 Firmware Update thanks Maarten for your great job!
> ------------------------------
> You can view, comment on, or merge this pull request online at:
>
>   https://github.com/maarten-pennings/CCS811/pull/58
> Commit Summary
>
>    - Add files via upload
>
> File Changes
>
>    - *A* Procedimiento para la actualización de firmwareCSS811_1.1.pdf
>    <https://github.com/maarten-pennings/CCS811/pull/58/files#diff-2ed3e88e5795cb2d2848bbd5d356e7f0108a43e07f1cb3cc0119897d1cab33a1>
>    (0)
>
> Patch Links:
>
>    - https://github.com/maarten-pennings/CCS811/pull/58.patch
>    - https://github.com/maarten-pennings/CCS811/pull/58.diff
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/maarten-pennings/CCS811/pull/58>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AHH3FQ3CLLQS75MUCZ32VWDSLVZD3ANCNFSM4SYGFYDA>
> .
>

erguro1973 commented:
Hi Maarten

Thank you so much for your great Job, I’m currently trying to fix baseline this sensor seems to be more complex to run it properly

> El 20 oct 2020, a las 15:32, Maarten Pennings <notifications@github.com> escribió:
> 
> ﻿
> Hi Ernesto,
> 
> Thanks, I have added the spanish doc.
> I did rename it slightly (underscores), and put it in a subdir, but I have
> also added a link from the main readme.
> 
> regards,
> 
> Maarten
> 
> 
> On Tue, Oct 20, 2020 at 1:38 PM Ernesto <notifications@github.com> wrote:
> 
> > Procedimiento para la actualización de firmwareCSS811_1.1.pdf
> > <https://github.com/erguro1973/CCS811/files/5407858/Procedimiento.para.la.actualizacion.de.firmwareCSS811_1.1.pdf>
> > Spanish guide for CSS811 Firmware Update thanks Maarten for your great job!
> > ------------------------------
> > You can view, comment on, or merge this pull request online at:
> >
> > https://github.com/maarten-pennings/CCS811/pull/58
> > Commit Summary
> >
> > - Add files via upload
> >
> > File Changes
> >
> > - *A* Procedimiento para la actualización de firmwareCSS811_1.1.pdf
> > <https://github.com/maarten-pennings/CCS811/pull/58/files#diff-2ed3e88e5795cb2d2848bbd5d356e7f0108a43e07f1cb3cc0119897d1cab33a1>
> > (0)
> >
> > Patch Links:
> >
> > - https://github.com/maarten-pennings/CCS811/pull/58.patch
> > - https://github.com/maarten-pennings/CCS811/pull/58.diff
> >
> > —
> > You are receiving this because you are subscribed to this thread.
> > Reply to this email directly, view it on GitHub
> > <https://github.com/maarten-pennings/CCS811/pull/58>, or unsubscribe
> > <https://github.com/notifications/unsubscribe-auth/AHH3FQ3CLLQS75MUCZ32VWDSLVZD3ANCNFSM4SYGFYDA>
> > .
> >
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or unsubscribe.
